cartography/intel/lastpass/users.py

```diff
@@ -0,0 +1,85 @@
+import logging
+from typing import Dict, List
+
+import neo4j
+from requests import Session
+from dateutil import parser as dt_parse
+
+from cartography.util import timeit
+
+logger = logging.getLogger(__name__)
+
+
+@timeit
+def sync(
+    neo4j_session: neo4j.Session,
+    update_tag: int,
+    lastpass_cid: str,
+    lastpass_provhash: str
+) -> None:
+    users = get(lastpass_cid, lastpass_provhash)
+    formated_users = transform(users)
+    load(neo4j_session, formated_users, update_tag)
+
+
+@timeit
+def get(lastpass_cid: str, lastpass_provhash: str) -> dict:
+    payload = {
+        'cid': lastpass_cid,
+        'provhash': lastpass_provhash,
+        'cmd': 'getuserdata',
+        'data': None
+    }
+    session = Session()
+    req = session.post('https://lastpass.com/enterpriseapi.php', data=payload, timeout=20)
+    req.raise_for_status()
+    return req.json()
+
+@timeit
+def transform(api_result: dict) -> List[Dict]:
+    result: List[dict] = []
+    for uid, user in api_result['Users'].items():
+        n_user = user.copy()
+        n_user['id'] = int(uid)
+        n_user['created'] = dt_parse.parse(user['created'])
+        n_user['last_pw_change'] = dt_parse.parse(user['last_pw_change'])
+        n_user['last_login'] = dt_parse.parse(user['last_login'])
+        result.append(n_user)
+    return result
+
+def load(
+    neo4j_session: neo4j.Session,
+    data: List[Dict],
+    update_tag: int,
+) -> None:
+
+    query = """
+    UNWIND $UserData as user
```

Comment:
non blocker: try to use our new method for automatically generating these queries.
https://lyft.github.io/cartography/dev/writing-intel-modules.html#load